membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -1,21 +1,22 @@
 package controllers
 
-import javax.inject._
-
-import actions.CommonActions
 import configuration.Config
-import models.ApiErrors.notFound
+import models.ApiError._
+import models.ApiErrors._
 import models.Fixtures
+import models.MembershipAttributes._
 import play.api.libs.concurrent.Execution.Implicits._
 import play.api.libs.json.Json
 import play.api.mvc.Results.Ok
-import play.api.mvc.{Action, Cookie}
-import services.{AttributeService, AuthenticationService}
+import play.api.mvc.{Action, Cookie, Result}
+import services.{AttributeService, AuthenticationService, DynamoAttributeService, IdentityAuthService}
 
 import scala.concurrent.Future
 
-class AttributeController @Inject() (attributeService: AttributeService) extends CommonActions {
+class AttributeController {
   val ADFREE_COOKIE_MAX_AGE = 60 * 60 * 6 // 6 hours
+  lazy val authenticationService: AuthenticationService = IdentityAuthService
+  lazy val attributeService: AttributeService = DynamoAttributeService
```

Comment:
Nope, the type on the LHS is not the same as in the RHS. If you omit the type the class will expect concrete services instead of traits.